app-typescript/components/ToggleBox/SimpleToggleBox.tsx

```diff
@@ -75,23 +79,9 @@ export class SimpleToggleBox extends React.PureComponent<IPropsSimple, IState> {
                     {badge ? badge : null}
                 </a>
                 <div className="toggle-box__content-wraper">
-                    {isOpen && !hideUsingCSS && (
-                        <div className="toggle-box__content" aria-describedby={`togglebox-${this.htmlId}`}>
-                            {children}
-                        </div>
-                    )}
-
-                    {hideUsingCSS && (
```

Comment:
if you're removing the implementation also drop other usages including in props and docs

Reply:
Done